yamlparse-applicative/src/YamlParse/Applicative/Pretty.hs

```diff
@@ -127,7 +128,7 @@ schemaDoc = go emptyComments
                         Just d -> blueOptional <+> ", default:" <+> pretty d
                     where blueOptional = annotate Blue "optional"
                in vsep
-                    [ keyDoc <> ":" <+> mkComment requiredDoc,
+                    [ annotate White keyDoc <> ":" <+> mkComment requiredDoc,
```

Comment:
This is not the only keyDoc, see the `<key>` stuff below.

Reply:
ah, I see. Done